app/views/default/data.html.erb

```diff
@@ -1,31 +1,46 @@
 <div class="row p-3">
     <div class="col-12">
-        <h1>Resources and Data on Systemic Racism</h1>
+        <h1>Resources & Data on Systemic Racism</h1>
         <p>Individual policies around housing and policing have systematic impacts, and when put together, these policies re-inforce and further propogate disparities.  We are gathering data to show the interconnectedness of these policies and their impact on systemic racism in the San Francisco Bay area and California.  Here are some searches to explore:</p>
-        <div class="row justify-content-center">
-            <div class="col-4 pb-4"><p class="red-banner">KNOW Systemic Racism Additional Resources</p>
-                <%=link_to image_tag('blm.png', alt: "Black Lives Matter"), "https://guides.library.stanford.edu/SystemicRacismTracker"%>
+        <div class="row gx-4 justify-content-center">
+            <div class="col-6 pb-4">
+              <div class="red-banner-body">
+                <p class="red-banner-title">KNOW Systemic Racism Additional Resources</p>
+                    <ul>
+                        <li class="pb-6 pt-4"><%=link_to "Library Guide", "https://guides.library.stanford.edu/SystemicRacismTracker"%></li>
+                        <li><%=link_to "Related projects happening on campus", "https://news.stanford.edu/2022/02/01/examining-systemic-racism-advancing-racial-equity/"%></li>
+                        <li><%=link_to "Say Their Names Spotlight Exhibit", "https://exhibits.stanford.edu/saytheirnames"%></li>
+                    </ul>
```

Comment:
link the headers (as well as the images), and then apply classes to hide the images on narrow width screens